pkg/sql/descriptor_mutation_test.go

```diff
@@ -324,6 +324,19 @@ CREATE TABLE t.test (k CHAR PRIMARY KEY, v CHAR, i CHAR DEFAULT 'i', FAMILY (k),
 			// Check that there are no hidden KV values for row "a",
 			// and column "i" for row "a" was deleted.
 			mTest.checkTableSize(afterDeleteKeys)
+
+			// Make column "i" a mutation.
+			mTest.writeColumnMutation("i", sqlbase.DescriptorMutation{State: state})
+			// Update primary key of row "c" to be "d"
+			mTest.Exec(`UPDATE t.test SET k = 'd' WHERE v = 'x'`)
+			// Make column "i" live so that it is read.
+			mTest.makeMutationsActive()
+			if state == sqlbase.DescriptorMutation_DELETE_ONLY {
+				mTest.CheckQueryResults(starQuery, [][]string{{"d", "x", "NULL"}})
+			} else {
+				mTest.CheckQueryResults(starQuery, [][]string{{"d", "x", "i"}})
+			}
+			mTest.checkTableSize(afterDeleteKeys)
```

Comment:
Can you move this entire test to above the DELETE test and just after UPSERT/UPDATE test above. That way it's in the right spot and you don't need to call checkTableSize twice.